In [ ]:
import pandas as pd

movies_full = pd.read_csv('ml-latest/movies.csv')

movies_full.shape, movies_full.columns

((86537, 3), Index(['movieId', 'title', 'genres'], dtype='str'))

Första jag kollar är hur stort datasetet är och får fram att datasetet innehåller **86537 rader** och **3 kolumner**.  
Kolumnerna är:  
- 'movieId'  (movie ID)
- 'title'  
- 'genres'

In [60]:
print(f'Antal duplicates i movieID: {movies_full['movieId'].duplicated().sum()}')
movies_full['title'].duplicated().value_counts()

Antal duplicates i movieID: 0


title
False    86330
True       207
Name: count, dtype: int64

Finns inga duplicates i 'movieId', men däremot så finns det 207 fall av duplicates i 'title'.